FirebasePerformance/Sources/FIRPerformance_Private.h

```diff
@@ -30,4 +31,7 @@
 /** Client object used for checking the status of the performance SDK before generating events. */
 @property(nonatomic, readwrite) FPRClient *fprClient;
 
+// Crashlytics instance initiated by interop.
+@property(nonatomic, nullable, strong) id<FIRCrashlyticsInterop> crashlytics;
```

Comment:
Also unnecessary for now

Reply:
Removed the changes to the private header.